### PR TITLE
Add Chainstack 2.1.2 release notes retrospectively 

### PR DESCRIPTION
**Pull request by neewy**

### CODE REVIEW

**Review by akegaviar — COMMENTED** [review]
We don't do x.x.x version relnotes. We only do x.x versions.

And looking at the changes introduced, I am not sure why we should start now.

**Comment by CaJIbHuK** [comment]
@neewy @easeev shall we close this PR?

**Comment by easeev** [comment]
> We don't do x.x.x version relnotes. We only do x.x versions.

Well, if we look at https://docs.chainstack.com/release-notes, we definitely did release notes for x.x.x :)

>@neewy @easeev shall we close this PR?

I have no strong opinion on that, IMO it makes sense to publish release notes when we have something to say. In this case there's some substance so it's fine to keep them

File: docs/release-notes.md [hunk 1]
```diff
@@ -8,6 +8,19 @@ meta:
...
+
+*November 2, 2020*
+
+### What's new
+
+* **Improved billing experience**. You can now reattempt past due payments through billing card change or directly in the activity log (where you can see more information attached to your billing events). 
+
+### Bug fixes
+
+* Hyperledger Fabric Explorer being out of sync with the network state.
```
Comment (easeev) [inline docs/release-notes.md:21]: Perhaps we should rephrase to make it clear that only some explorers were out of sync and not all

Comment (neewy) [inline docs/release-notes.md:21]:
```suggestion
* Resolved an issue with some Hyperledger Fabric Explorers being out of sync with the network state.
```

File: docs/release-notes.md [hunk 2]
```diff
@@ -8,6 +8,19 @@ meta:
 
 # Release notes
 
+## Chainstack 2.1.2
+
+*November 2, 2020*
+
+### What's new
+
+* **Improved billing experience**. You can now reattempt past due payments through billing card change or directly in the activity log (where you can see more information attached to your billing events).
```
Comment (akegaviar) [inline docs/release-notes.md:17]: Change to **Billing**.  We put component name in bold and then expand on what's done. The component here is billing (also mentioned in previous releases)

Comment (akegaviar) [reply docs/release-notes.md:17]: Something like this:

```
You can now reattempt past due payments through billing card change or directly in the activity log. The activity log also provides more information on your billing events.
```

Please avoid brackets as much as possible.

**Review by akegaviar — CHANGES_REQUESTED** [review]

File: docs/release-notes.md (hunk 2 above)
Comment (neewy) [inline docs/release-notes.md:17]:
```suggestion
* **Billing**. You can now reattempt past due payments through billing card change or directly in the activity log. The activity log also provides more information on your billing events. 
```

**Review by akegaviar — DISMISSED** [review]

**Comment by neewy** [comment]
I can't merge without second approve, whenever you have a sec @CaJIbHuK please check it 🙏🏻

File: docs/release-notes.md [hunk 3]
```diff
@@ -8,6 +8,19 @@ meta:
...
+
+*November 2, 2020*
+
+### What's new
+
+* **Billing**. You can now reattempt past due payments through billing card change or directly in the activity log. The activity log also provides more information on your billing events. 
+
+### Bug fixes
+
+* Resolved an issue with some Hyperledger Fabric Explorers being out of sync with the network state.
```
Comment (easeev) [inline docs/release-notes.md:21]: BTW there's no such thing as "Hyperledger Fabric Explorer", there's "[Hyperledger Explorer](https://github.com/hyperledger/blockchain-explorer)" or perhaps "Hyperledger Fabric explorer" if you want to mention Fabric

Comment (neewy) [inline docs/release-notes.md:21]:
```suggestion
* Resolved an issue with some Hyperledger Explorer instances in Fabric networks being out of sync with the blockchain state.
```

**Review by easeev — APPROVED** [review]
LGTM

**Review by akegaviar — APPROVED** [review]

**Pull request merged**